frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelinesList.cy.ts

```diff
@@ -66,9 +94,629 @@ describe('PipelinesList', () => {
+    configurePipelineServerModal.findEndpointInput().type('https://s3.amazonaws.com/');
+    configurePipelineServerModal.findRegionInput().should('have.value', 'us-east-1');
+    configurePipelineServerModal.findBucketInput().type('test-bucket');
+    configurePipelineServerModal.findSubmitButton().should('be.enabled');
+    configurePipelineServerModal.findSubmitButton().click();
+
+    cy.wait('@createSecret').then((interception) => {
+      expect(interception.request.url).to.include('?dryRun=All');
+      expect(interception.request.body).to.containSubset({
+        metadata: {
+          name: 'dashboard-dspa-secret',
+          namespace: projectName,
+          annotations: {},
+          labels: { 'opendatahub.io/dashboard': 'true' },
+        },
+        stringData: { AWS_ACCESS_KEY_ID: 'test-aws-key', AWS_SECRET_ACCESS_KEY: 'test-secret-key' },
+      });
+    });
+    cy.wait('@createSecret').then((interception) => {
+      expect(interception.request.url).not.to.include('?dryRun=All');
+    });
+
+    cy.get('@createSecret.all').then((interceptions) => {
+      expect(interceptions).to.have.length(2); // 1 dry-run request and 1 actual request
+    });
+
+    cy.wait('@createDSPA').then((interception) => {
+      expect(interception.request.body).to.containSubset({
+        metadata: { name: 'dspa', namespace: projectName },
+        spec: {
+          apiServer: { enableSamplePipeline: false },
+          dspVersion: 'v2',
+          objectStorage: {
+            externalStorage: {
+              host: 's3.us-east-1.amazonaws.com',
+              scheme: 'https',
+              bucket: 'test-bucket',
+              region: 'us-east-1',
+              s3CredentialsSecret: {
+                accessKey: 'AWS_ACCESS_KEY_ID',
+                secretKey: 'AWS_SECRET_ACCESS_KEY',
+                secretName: 'test-secret',
+              },
+            },
+          },
+        },
+      });
+    });
+  });
+
+  it('should configuring pipeline server to connect external DB', () => {
+    initIntercepts();
+    projectDetails.visitSection(projectName, 'pipelines-projects');
+
+    cy.interceptK8sList(
+      DataSciencePipelineApplicationModel,
+      mockK8sResourceList([mockDataSciencePipelineApplicationK8sResource({})]),
+    );
+
+    cy.interceptK8s(
+      DataSciencePipelineApplicationModel,
+      mockDataSciencePipelineApplicationK8sResource({}),
+    );
+
+    cy.interceptK8s(
+      'POST',
+      {
+        model: SecretModel,
+        ns: projectName,
+      },
+      mockSecretK8sResource({ namespace: projectName }),
+    ).as('createSecret');
+
+    cy.interceptK8s(
+      'POST',
+      DataSciencePipelineApplicationModel,
+      mockDataSciencePipelineApplicationK8sResource({
+        namespace: projectName,
+      }),
+    ).as('createDSPA');
+
+    pipelinesGlobal.findConfigurePipelineServerButton().should('be.enabled');
+    pipelinesGlobal.findConfigurePipelineServerButton().click();
```

Comment:
Please update `pipelinesGlobal` to `pipelinesSection` here and in all other places.